src/module.ts

```diff
@@ -35,6 +36,27 @@ export default defineNuxtModule<SwiperModuleOptions>({
         from: 'swiper/vue'
       }
     ]
+    
+    // Separating Swiper into chunks from Nuxt entry bundle
+    nuxt.options.vite = defu(
+      {
+        build: {
+          rollupOptions: {
+            manualChunks(id) {
+              if (id.includes('/node_modules/')) {
+                const modules = ['swiper']
+                const chunk = modules.find((module) => id.includes(`/node_modules/${module}`))
+
+                if (chunk) {
+                  return `vendor-${chunk}`
+                }
+              }
```

Comment:
Couldn't it be possible to be a one line if since we aren't worried about multiple dependancies ?

```suggestion
              if (id.includes('/node_modules/swiper')) {
                  return `vendor-swiper`
              }
```

Reply:
Sure